libsignal-service/src/pre_keys.rs

```diff
@@ -180,7 +190,7 @@ pub(crate) const PRE_KEY_MINIMUM: u32 = 10;
 pub(crate) const PRE_KEY_BATCH_SIZE: u32 = 100;
 pub(crate) const PRE_KEY_MEDIUM_MAX_VALUE: u32 = 0xFFFFFF;
 
-pub(crate) async fn replenish_pre_keys<
+pub async fn replenish_pre_keys<
     R: rand::Rng + rand::CryptoRng,
```

Comment:
I'm not necessarily convinced we want this exposed outside of the crate.

Reply:
I'll check, might be a remnant, but it might also be me being lazy with the current state of the linking code between this and `presage`.

Reply:
Restored!